graylog2-web-interface/src/views/components/visualizations/number/AutoFontSizer.jsx

```diff
@@ -16,45 +16,51 @@ type Props = {
 };
 
 const _multiplierForElement = (element, targetWidth, targetHeight) => {
+  const childSizeRatio = 0.8; // Propotion of the child size in relation to the container
   const contentWidth = element.offsetWidth;
   const contentHeight = element.offsetHeight;
 
-  const widthMultiplier = (targetWidth * 0.8) / contentWidth;
-  const heightMultiplier = (targetHeight * 0.8) / contentHeight;
+  const widthMultiplier = (targetWidth * childSizeRatio) / contentWidth;
+  const heightMultiplier = (targetHeight * childSizeRatio) / contentHeight;
   return Math.min(widthMultiplier, heightMultiplier);
 };
 
-const AutoFontSizer = ({ children, target, height, width }: Props) => {
+const _updateFontSize = (setFontSize, currentSize, newSize) => {
+  if (currentSize !== newSize && newSize !== 0 && Number.isFinite(newSize)) {
+    setFontSize(newSize);
+  }
+};
+
+const useAutoFontSize = (target, _container, height, width) => {
+  // This hook will update the font size based on the difference between the dimensions of the container and its child.
+  // The font size is being recalculated, until the mentioned difference is within the tolerance.
+  const tolerance = 0.05;
```

Comment:
This constant could be pulled out of the function scope and into the module scope.